py3-azure-storage-blob.yaml

```diff
@@ -0,0 +1,49 @@
+# Generated from https://pypi.org/project/azure-storage-blob/
+package:
+  name: py3-azure-storage-blob
+  version: 12.19.0
+  epoch: 0
+  description: Microsoft Azure Blob Storage Client Library for Python
+  copyright:
+    - license: MIT License
+  dependencies:
+    runtime:
+      - py3-azure-core
+      - py3-cryptography
+      - py3-typing-extensions
+      - py3-isodate
+      - python3
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - py3-setuptools
+      - py3-wheel
+      - python3
+      - python3-dev
+      - wolfi-base
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 997948617b7226f492c4829fba31ba2f72ccc1f1
+      repository: https://github.com/Azure/azure-sdk-for-python
+      tag: azure-storage-blob_${{package.version}}
+
+  - runs: |
+      cd sdk/storage/azure-storage-blob
+      python setup.py build
+      python setup.py install --prefix=/usr --root="${{targets.destdir}}"
+
+  - uses: strip
+
+update:
+  enabled: true
+  manual: false
```

Comment:
QQ - what does having enabled: true and manual:false do here?

I'm asking as i'm not so sure - vs a suggestion to change